planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/utils/utils.hpp

```diff
@@ -60,15 +61,9 @@ using geometry_msgs::msg::Pose;
 using geometry_msgs::msg::Twist;
 using path_safety_checker::CollisionCheckDebugMap;
 using tier4_planning_msgs::msg::PathWithLaneId;
-
+rclcpp::Logger get_logger();
```

Comment:
do you need to declare it in the header?

Reply:
I want dont want to re-use the logger in utils/path, so this have to be in the header.

Reply:
i removed the get_logger from utils.hpp. instead, for the sake of standardizing our code, I've used exception